README.md

```diff
@@ -1,2 +1,25 @@
-# Hey-Micro
-My Hello Microverse Project
+# Hello-Micrverse
+
+This is my Hello Microverse project.
+
+### Built With
+Html<br />
+CSS
+
+### AUTHOR
+
+Inyambo Situmbeko
+
+<a href="https://github.com/BlakeYall">Github</a> 
+
+<a href="https://twitter.com/Thee_indigo1">Twitter</a> 
+
+<a href="https://www.linkedin.com/in/inyambo-situmbeko-524bb7229/">Linkedin</a>
+
+### Acknowledgments
+Thank you to Microverse
+
+### License
+This is an MIT licensed project
```

Comment:
- [ ] Please link your license file here :+1: to make it easily accessible for everyone. You can do this writing something like this **This is an [MIT](./LICENSE) licensed project.** :blush: